system/Database/BaseBuilder.php

```diff
@@ -2000,7 +2000,7 @@ protected function _upsertBatch(string $table, array $keys, array $values): stri
     /**
      * Set table alias for dataset pseudo table.
      */
-    private function setAlias(string $alias): BaseBuilder
+    public function setAlias(string $alias): BaseBuilder
```

Comment:
Please revert. This method is not a public method.
We can set the alias with `setData()` public method.